src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Pages/_ViewImports.cshtml

```diff
@@ -1,3 +1,2 @@
-@using RazorComponentsWeb_CSharp.Components
 @namespace RazorComponentsWeb_CSharp.Pages
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
Could we inline the `_ViewImports.cshtml` into `Host.cshtml` so there's no need for a separate `_ViewImports.cshtml` file?

This will reduce the chance of anyone being confused about which file to put their global `@using` (etc) directives into.

I know this will be slightly less inconvenient for people who want to add more `.cshtml` files, but it optimises for the pure Razor Components case, which is the whole purpose of this template.

@danroth27 / @rynowak - Your opinions also welcomed on this.

Reply:
I’m happy with it as long as @danroth27 is happy.

Reply:
I'm happy with that

Reply:
ok

Reply:
Done